src/pages/workspaces/workspace/Analyses.js

```diff
@@ -334,11 +334,9 @@ const Analyses = _.flow(
   const noAnalysisBanner = div([
     div({ style: { fontSize: 48 } }, ['A place for all your analyses ']),
     div({ style: { display: 'flex', flexDirection: 'row', justifyContent: 'space-evenly', alignItems: 'center', columnGap: '5rem' } }, _.dropRight(!!googleProject ? 0 : 2, [
-      img({ src: jupyterLogo, style: { height: 120, width: 80 } }),
-      img({ src: rstudioBioLogo, style: { width: 400 } }),
-      div([
-        img({ src: galaxyLogo, style: { height: 60, width: 208 } })
-      ])
+      img({ src: jupyterLogo, style: { height: 120, width: 80 }, alt: 'Jupyter' }),
+      img({ src: rstudioBioLogo, style: { width: 400 }, alt: 'RStudio Bioconductor' }),
+      img({ src: galaxyLogo, style: { height: 60, width: 208 }, alt: 'Galaxy' })
```

Comment:
If you haven't already, check with @jdcanas about why he wrapped the Galaxy image in a div.

Reply:
Good idea - thank you! 
I removed it and saw it render the same, so I'm curious what his thoughts are.